### DISCUSSION TITLE
Workflow Fire&Forget from Http Endpoint

### DISCUSSION THREAD

**Discussion by tolgaevcimen** [elsa-workflows/elsa-core#2461] [Q&A]
I want my long running workflow to be triggered from an [http endpoint](https://github.com/elsa-workflows/elsa-core/blob/master/src/activities/Elsa.Activities.Http/Activities/HttpEndpoint/HttpEndpoint.cs) with a fire & forget behavior. [Write Http response](https://github.com/elsa-workflows/elsa-core/blob/master/src/activities/Elsa.Activities.Http/Activities/WriteHttpResponse/WriteHttpResponse.cs) just waits if I put a fork between the long running branch and http response activity for the long running branch to complete. 

What is the simplest way to handle this use case?

**Comment by sfmskywalker** [comment]
I just tried that construct, and it works well when I run the following workflow:

![image](https://user-images.githubusercontent.com/938393/142937996-a40cd94d-d3cc-4936-8428-cf113721ee38.png)

When I trigger the workflow, I see an immediate HTTP response in my browser, and 15 seconds later a message in my server's console window.

[workflow-definition-7448d8d2ded64c868b474b8afce9e852.zip](https://github.com/elsa-workflows/elsa-core/files/7584505/workflow-definition-7448d8d2ded64c868b474b8afce9e852.zip)

Perhaps if you share your workflow I can see what might be the issue.
